.gitignore

```diff
@@ -1,5 +1,8 @@
 #### GENERAL DEVELOPMENT
 
+# Top-level hidden files/folders
+.*
```

Comment:
Could you please elaborate what you're hiding ?

Reply:
Good point.  I've changed it to hide only hidden folders.  I use hidden folders for my own temporary test files (e.g. blueprints, deployment files, etc.).

Reply:
You can create your own .gitignore using
```sh
git config --global core.excludesfile $HOME/.gitignore
```

Reply:
Cool, didn't know that.  Thanks!

Reverted the `.gitignore` changes.